dist/OBS-statistics.R

```diff
@@ -0,0 +1,291 @@
+# We use ggplot2, a library that makes painting graphics much easier.
+# You can check the types of graphs and examples code here: https://plot.ly/ggplot2
+
+# REQUIRE NEEDED LIBRARIES, in case that you don't have then installed, use: install.packages("ggplot2")
```

Comment:
Was used for the presentation, but can also be used to take statistics just to know what is going in any instance of OBS. I tried to documented the R file a lot so it can be easily used by everybody.

R is free software too, so I don't see the problem in including the file. Also, the rake task do not make sense without a way to visualise them. And may be useful for other people too.

@openSUSE/open-build-service opinions?